src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -25,12 +25,12 @@
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System.Core" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot' or '$(TargetGroup)' == 'netcoreapp1.2corert'">
     <Compile Include="System\Threading\Tasks\TaskExtensions.CoreRT.cs" />
```

Comment:
BTW: It would be nice to just move this to CoreLib when we get a chance. There is no point to having it in corefx because of the implementation is completely different between CoreCLR and CoreRT anyway, and so there is no sharing anyway. Also, the implementation is slower than necessary because of this split.